### PR TITLE
Update copy for WASD Summer

### PR DESCRIPTION
**Pull request by joshdavies14**
This PR contains updated copy ahead of WASD Summer in August (copied over from Discord).

The countdown should also be up to date (start time may need changing?), and currently it says signups aren't open, but we can change that when we go live. :)

### CODE REVIEW

File: content/whats_on.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 <div class="content" markdown="1">
+
 #### Weekend-Long Speedruning Stream
-![A speedrunner and his co-commentator at the event](http://localhost:8080/dist/img/speedrunning.jpg)
+![A speedrunner and their co-commentator at the event](http://localhost:8080/dist/img/speedrunning.jpg)
```
Comment (davidjrichardson) [inline content/whats_on.md]: MD: This title is incorrect for the image shown - it's a wide angle shot of the event.

File: content/whats_on.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+Two days of speedrunning in aid of [SpecialEffect](https://specialeffect.org) streamed live on the WASD [Twitch channel](https://twitch.tv/warwickspeedrun). Come and say hi in the chat!
 </div>
 
 <div class="content" markdown="1">
+
 #### Welcoming Community
 ![Spectators enjoying themselves at the event](http://localhost:8080/dist/img/community.jpg)
 
-Our events have a wonderful sense of community and we welcome everyone with open arms.
+Our events have a wonderful sense of community, and we welcome everyone with open arms!
```
Comment (davidjrichardson) [inline content/whats_on.md]: ??: Why did we add this comma?

File: templates/template.html [hunk 3]
```diff
@@ -43,11 +43,11 @@
   <meta property="og:image" content="http://localhost:8080/dist/img/og-avatar.png">
   <meta property="og:image:width" content="400px">
   <meta property="og:image:height" content="400px">
-  <meta property="og:title" content="Warwick's Awesome Speedruns and Demos 2020">
+  <meta property="og:title" content="Warwick's Awesome Speedruns and Demos 2021">
```
Comment (davidjrichardson) [inline templates/template.html]: MD: This should be changed to reflect the event title proper

File: templates/template.html [hunk 4]
```diff
@@ -166,15 +166,16 @@ <h1 class="title is-size-2">Event Info</h1>
     <h1 class="title is-size-2">What's On</h1>
     <hr class="title-divider"/>
     <div class="content">
-      <p>We hope to run WASD2021 as an in-person event but that may not be possible because of COVID. The following is what we will have available if a physical event can be safely run:</p>
+      <p>Due to the ongoing situation, we can't ensure upcoming events will be run in-person. Our next event, WASD Summer, will be entirely remote, with the below available. We hope to be able to run the next WASD event in person!</p>
```
Comment (davidjrichardson) [inline templates/template.html]: SD: The sentence starting "Our next event..." really should be re-worded IMO; three commas is too much. I also don't quite understand what "with the below available" refers to.

**Review by davidjrichardson — CHANGES_REQUESTED** [review]

File: templates/template.html (hunk 3 above)
Comment (joshdavies14) [reply templates/template.html]: Fixed in new commit

File: content/whats_on.md (hunk 1 above)
Comment (joshdavies14) [reply content/whats_on.md]: Fixed in new commit

File: content/whats_on.md (hunk 2 above)
Comment (joshdavies14) [reply content/whats_on.md]: Fixed in new commit, PyCharm inserted this one...

File: templates/template.html (hunk 4 above)
Comment (joshdavies14) [reply templates/template.html]: Fixed in new commit

File: templates/template.html [hunk 5]
```diff
@@ -8,15 +8,15 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="theme-color" content="#8338EC"/>
 
-  <title>Warwick's Awesome Speedruns and Demos 2021</title>
+  <title>Warwick's Awesome Speedruns and Demos Summer</title>
```
Comment (davidjrichardson) [inline templates/template.html]: MD: We should indicate what year WASD Summer is for if it's going to be a regular event like the main one in February. This goes for all of the places we mention WASD Summer.

**Review by davidjrichardson — CHANGES_REQUESTED** [review]
One small thing about including the year of the event in the title but it looks good to me now :+1:

File: templates/template.html (hunk 5 above)
Comment (joshdavies14) [reply templates/template.html]: Thanks! Should now all be up to date, lmk if I missed a place :)

File: content/event_info.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
 
-Donations to the event are in support of [SpecialEffect](https://specialeffect.org.uk/), a wonderful UK-based charity whose mission is to put fun and inclusion back into the lives of people with physical disabilities, by helping them to play video games. They do this by using and developing a range of accessibility hardware and software, from modified joypads to eye-gaze technology. The WASD events to date have raised over £5,000 for SpecialEffect!
+Donations to the event are in support of [SpecialEffect](https://specialeffect.org.uk/), a wonderful UK-based charity whose mission is to put fun and inclusion back into the lives of people with physical disabilities, by helping them to play video games. They achieve this by using and developing a range of accessibility hardware and software, from modified joypads to eye-gaze technology. To date, the WASD events have raised over £5,000 for SpecialEffect!
```
Comment (davidjrichardson) [inline content/event_info.md:6]: SD: I am not sure about the amount now and it's still _technically_ £5k+ but I think we're above £6k raised now. It's not an issue though.

**Review by davidjrichardson — APPROVED** [review]
LGTM :+1:

**Pull request merged**